### ISSUE TITLE
Running TREX when samples has been divided into several FASTQ files

### ISSUE THREAD

**Issue by acorbat** [frisen-lab/TREX#69]
I was going through how to run TREX when a single sample was separated into several files. Although it seems to identify every bam file in a folder and put the reads together (#27), there is a single file of cell ids that can be used for this. Although it might be unlikely, it could happen that an allowed cell id for one dataset, should not be allowed in another dataset. I believe that two paired lists of paths and allowed cell id files could be provided to avoid this problem. Is there some other workaround? Am I missing something?

**Comment by marcelm** [comment]
I haven’t used the command with multiple samples so far, so I had to test this a bit.

To clarify (mostly to myself): One doesn’t specify multiple BAM files directly, but multiple 10x output folders like this:
```
trex run10x folder1 folder2
```
For each sample, TREX then uses the BAM file in `.../outs/possorted_genome_bam.bam` and the file with the list of allowed cell IDs in `.../outs/barcodes.tsv(.gz)`.

So this list of allowed cell IDs is already sample-specific. However, I guess this is not what you meant. Are you referring to the `--filter-cellids` option? Then I believe it is correct that there’s no way to specify this on a per-sample basis.

**Comment by acorbat** [comment]
Yes, I meant the ```--filter-cellids``` option. But now that you mention this, it's probably easier to add/update the ```.../outs/barcodes.tsv``` file with the allowed cell id file, and not provide the ```--filter-cellids``` parameter. I can't imagine why this would not work. Do you foresee any differences or problems? Also, I don't understand why there are two different ways of filtering cell ids.

**Comment by acorbat** [comment]
For the time being, I am running it the "unclean" way.

I will leave the issue open as it should probably be dealt with.
